packages/modelviewer.dev/examples/wcag/index.html

```diff
@@ -0,0 +1,168 @@
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>&lt;model-viewer&gt; WCAG</title>
+    <meta charset="utf-8" />
+    <meta name="description" content="&lt;model-viewer&gt; WCAG examples" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link type="text/css" href="../../styles/examples.css" rel="stylesheet" />
+    <link type="text/css" href="../../styles/docs.css" rel="stylesheet" />
+    <link
+      rel="shortcut icon"
+      type="image/png"
+      href="../../assets/favicon.png"
+    />
+
+    <script defer src="https://web3dsurvey.com/collector.js"></script>
+    <script>
+      window.ga =
+        window.ga ||
+        function () {
+          (ga.q = ga.q || []).push(arguments);
+        };
+      ga.l = +new Date();
+      ga("create", "UA-169901325-1", { storage: "none" });
+      ga("set", "referrer", document.referrer.split("?")[0]);
+      ga("set", "anonymizeIp", true);
+      ga("send", "pageview");
+    </script>
+    <script async src="https://www.google-analytics.com/analytics.js"></script>
+  </head>
+
+  <body>
+    <div class="examples-page">
+      <div class="sidebar" id="sidenav"></div>
+      <div id="toggle"></div>
+
+      <div class="examples-container">
+        <div class="sample">
+          <div id="useA11y" class="demo"></div>
+          <div class="content">
+            <div class="wrapper">
+              <h4 id="intro">
+                <span class="font-medium">Use a11y. </span>Improve
+                accessibility, by adding a11y attributes to the model-viewer
+                element. To have more descriptive translations to improve screen
+                reader capability. .
+              </h4>
+
+              <div class="heading">
+                <h2 class="demo-title">A11y defined translations</h2>
+                <h4></h4>
+              </div>
+              <example-snippet stamp-to="useA11y" highlight-as="html">
+                <template>
+                  <model-viewer
+                    id="reveal"
+                    loading="eager"
+                    camera-controls
+                    touch-action="pan-y"
+                    auto-rotate
+                    src="../../shared-assets/models/Astronaut.glb"
+                    shadow-intensity="1"
+                    alt="A 3D model of a shishkebab"
+                    a11y="../../assets/a11y.json"
```

Comment:
Can you expand this example to demonstrate changing the locality dynamically, since that's probably what any user would be trying to do?